composables/useCards.ts

```diff
@@ -0,0 +1,50 @@
+import type { Card } from "~/types/models/Card";
+
+export function useCards() {
+  const cards = ref<Card[]>();
+  const currentPage = ref(1);
+  const totalPages = ref(1);
```

Comment:
save the whole response on the cards

so you wont need new variable for current page and total pages
check how we;re doing it to the uaap project

Reply:
Already create Pagination Interface and added it to cards variable